### DISCUSSION TITLE
Add shared subscription strategy

### DISCUSSION THREAD

**Discussion by alexa22013** [emqx/emqx#9554] [Q&A]
I want to use shared subscription "strategy = sticky" for a group. Tried both of the below in config but none is working, the broker is still using round robin. 

```
shared_subscription_group.test {
    strategy = sticky
}
```
```
shared_subscription_strategy = sticky
```
Am I doing something wrong? 
Please help. Thanks in advance...

**Comment by zmstone** [comment]
Hi @alexa22013 
which version are you using?
